client/src/containers/AppContainer.tsx

```diff
@@ -1,12 +1,15 @@
 import { connect } from 'react-redux';
 import App, { AppProps } from '../components/App';
 import { State } from '../reducers/index';
+import { loginCheck } from '../actions/user';
+
 
 function mapStateToProps(state: State): AppProps {
     return {isLoggedIn: state.session.uname !== undefined};
 }
 
-function mapDispatchToProps(_dispatch: any): {} {
+function mapDispatchToProps(dispatch: any): {} {
+    dispatch(loginCheck());
```

Comment:
Let's do this in index.tsx before first render. Ideally there'll be all sorts of logic for recovering the initial state.